### ISSUE TITLE
[PROPOSED WORK ITEM] Verifiable Request Data Model

### ISSUE THREAD

**Issue by glcohen** [w3c-ccg/community#179]
# New Work Item Proposal

See [W3C-CCG New Work Item Process](https://docs.google.com/document/d/1vj811aUbs8GwZUNo-LIFBHafsz4rZTSnRtPv7RQaqNc/edit#)

## Include Link to Abstract or Draft 

https://docs.google.com/presentation/d/16pcCj1VtJ0r2OcLM1GznpDreWSgedeBAB8lZtLlkIYM/edit?usp=sharing

## List Owners

> Identify 1 lead (person responsible for advancing the work item) and at least 1 other owner. Ideally, include their github usernames

Lead: @glcohen 
Other owner: <your-name-here!>

**Comment by dlongley** [comment]
We do have a work item around verifiable requests here: https://w3c-ccg.github.io/vp-request-spec/

Perhaps the efforts could be joined?

**Comment by glcohen** [comment]
@dlongley thanks. I'm open to joining efforts with that work item if you, Mike Varley, and @dmitrizagidulin agree it's a good fit.

**Comment by wyc** [comment]
Looping in @clehner who may be interested in participation.

**Comment by kimdhamilton** [comment]
I'd like to get in on this. Github username is @kimdhamilton

**Comment by glcohen** [comment]
It doesn't look like I'll have the time I wish I had to see this through. @kimdhamilton @dlongley I'd love to hand over the reigns to one of you and help from the sidelines

**Comment by kimdhamilton** [comment]
I don't have the bandwidth to be the primary driver of it, at least not until this becomes a more pressing issue for me. Unless we have a "champion", we may need to put this on hold (which from a chair perspective means closing this issue, which one can reopen when it becomes more actionable)

**Comment by vsnt** [comment]
There is currently no owner for this work item. This was brought up at 2/10/2021 meeting, there were no volunteers. The Chairs will give 14 days to find 2 owners, and if there are no owners by then, will close this item. It can be re-opened later when there is more interest. 

Close item on 2/25/2021 if unable to have 2 owners by 2/24/2021.

**Comment by vsnt** [comment]
No owners volunteered, closing item.
